### PR TITLE
Publication updates

### PR DESCRIPTION
**Pull request by rosibaj**

### CODE REVIEW

File: docs/data-access/publication-guidelines.md [hunk 1]
```diff
@@ -5,8 +5,10 @@ title: Publication Guidelines
 
 The official ARGO [Publication Policy](https://www.icgc-argo.org/page/77/e3-publication-policy) can be found on the ICGC ARGO website.
 
-To cite the ARGO Data Platform, please cite these publications:
+If you are looking for a way to cite the ARGO Data Platform as a whole, please cite the landing page (http://platform.icgc-argo.org) as you would any website. We are planning to write a manuscript about the ARGO Data Platform, and once that is published or posted as a pre-print, we will update this article with full citation information.
```
Comment (kcullion) [inline docs/data-access/publication-guidelines.md:8]: - [ ] maybe say please cite the `Platform url` instead of the landing page?

- [ ] to make it sound like it's coming...would reword to:

`We are currently writing a manuscript about the ARGO Data Platform, and once that is published or posted as a pre-print, we will update this page with full citation information.`

Comment (rosibaj) [reply docs/data-access/publication-guidelines.md:8]: - landing page requested by christina - this is more correct bc technically ther are many urls, but we mean the homepage
- we arent writing it yet :) lets leave that as it

Comment (kcullion) [reply docs/data-access/publication-guidelines.md:8]: k maybe just say "page" instead of "article"
